projects/vdk-control-cli/src/vdk/internal/control/configuration/vdk_config.py

```diff
@@ -208,7 +209,25 @@ def read_configuration(self, section, option, fallback=None):
             log.debug(f"Reading configuration file: {configuration_file_path} ...")
 
             config_parser = ConfigParser()
-            config_parser.read(configuration_file_path)
+            try:
+                config_parser.read(configuration_file_path)
+            except (MissingSectionHeaderError, Exception) as e:
+                log.debug(e, exc_info=log.getEffectiveLevel() == logging.DEBUG)
```

Comment:
What is this ? log.getEffectiveLevel() == logging.DEBUG seems unnecessary since log.debug ...

Reply:
Good catch. This was meant for log error, not debug. Fixed, thanks